### PR TITLE
Reframe 3.0 - fix package bug

### PR DESCRIPTION
**Pull request by victorusu**
I have forgotten to add a `self.` before on the `spec.version` call on line 60. 
This fixes the error when attempting to run
```
spack install --fake reframe
```

The error message is
```
reframe: Building reframe [Package]
==> Error: NameError: name 'spec' is not defined

/users/hvictor/spack-victor/var/spack/repos/builtin/packages/reframe/package.py:60, in setup_run_environment:
         59    def setup_run_environment(self, env):
  >>     60        if spec.version >= Version('3.0'):
         61            if "+docs" in spec:
         62                env.prepend_path('MANPATH',  self.prefix.docs.man)
```

### CODE REVIEW

File: var/spack/repos/builtin/packages/reframe/package.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class Reframe(Package):
     maintainers = ['victorusu', 'vkarak']
 
     version('master',   branch='master')
-    version('3.0',       sha256='c4fe84a92d961546e4d0e33ca3476ba0d4cebc908eb4e33897f646bd1fd5205b')
+    version('3.0',       sha256='fb76b4513c03b84f5b3bbbc988f7747e5b58f04c983b3935bab1f2e81adccb82')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/reframe/package.py:26]: Why this change in `sha256`?

**Review by alalazo — COMMENTED** [review]
Just a minor question, otherwise LGTM

File: var/spack/repos/builtin/packages/reframe/package.py (hunk 1 above)
Comment (vkarak) [reply var/spack/repos/builtin/packages/reframe/package.py:26]: For some strange reason, also the current version of the package in the develop branch has the wrong hash and does not install.

Comment (victorusu) [reply var/spack/repos/builtin/packages/reframe/package.py:26]: When making the recipe I used another version of ReFrame because the 3.0 was not released yet. I forgot to update the checksum to point to the correct one when ReFrame 3.0 was released.

**Review by alalazo — APPROVED** [review]
